e2e_tests/tests/nightly/test_distributed.py

```diff
@@ -282,8 +282,21 @@ def test_remote_search_runner() -> None:
     exp.run_basic_test(config, conf.custom_search_method_examples_path("asha_search_method"), 1)
 
 
+HUGGINGFACE_CONTEXT_ERR_MSG = """
+HF_READ_ONLY_TOKEN environment variable is missing!
+
+If this error was raised in CircleCI, verify that the test was run using the
+`hugging-face` context. This context should be injecting the expected
+`HF_READ_ONLY_TOKEN`variable.
+
+If this error was raised in another context, set the HF_READ_ONLY_TOKEN env var manually with its
+value a valid HF authorization token.
+"""
+
+
 @pytest.mark.distributed
```

Comment:
I suspect you wanted to drop this decoration, else maybe it'll be run in both workflows?

Reply:
I did't think that's how it works?  I was trying to follow the [deepspeed example](https://github.com/determined-ai/determined/blob/6d4e9f039fe6e2c54dd202edb0598aa4d5b48a52/e2e_tests/tests/nightly/test_distributed.py#L385-L387) which seems to use all applicable marks.

Reply:
nod -- they're running as a part of both workflows. Here's an execution of `test-e2e-gpu-distributed` that runs `tests/nightly/test_distributed.py::test_hf_trainer_image_classification_deepspeed_autotuning`:  https://app.circleci.com/pipelines/gh/determined-ai/determined/44291/workflows/ba96ce4c-64bd-492c-9506-58d99287a4ac/jobs/1795361/parallel-runs/0/steps/0-129

Reply:
[Here](https://github.com/determined-ai/determined/blob/16a62627db2e190da03968a90a20ece955576f0a/.circleci/config.yml#L3079)'s an example of a CircleCI workflow using multiple marks:

```yaml
      - test-e2e-aws:
          name: test-e2e-cpu-distributed
            parameters:
              mark: ["distributed and not gpu_required"]
```

Reply:
Removed the `huggingface` mark and just lumped the tests with `distributed`